pkg/network/tracer/compile.go

```diff
@@ -32,5 +32,12 @@ func getCFlags(config *config.Config) []string {
 	if config.BPFDebug {
 		cflags = append(cflags, "-DDEBUG=1")
 	}
+	if config.EBPFInstrumentationEnabled {
```

Comment:
please add a comment (you can copy paste from PR description) to explain the purpose of this flag

Reply:
I have described the config option where it is defined
https://github.com/DataDog/datadog-agent/pull/23070/files#diff-7a70d1343f42219c3b2c07d3bf90635f3d79b9e4fa7ed18ede433218e25e929eR82-R83